metadata-ingestion/src/datahub/integrations/great_expectations/action.py

```diff
@@ -317,9 +317,9 @@ def get_assertions_with_results(
                     type=AssertionResultType.SUCCESS
                     if success
                     else AssertionResultType.FAILURE,
-                    rowCount=result.get("element_count"),
-                    missingCount=result.get("missing_count"),
-                    unexpectedCount=result.get("unexpected_count"),
+                    rowCount=None if result.get("element_count") is None else round(result.get("element_count")),
```

Comment:
I believe using `int()` would be more correct over using `round()`, although both would yield same result as this count would never have nonzero digit after decimal point.

Reply:
Yes, your are sure! I Modify this code. Thank you for your suggestion.